src/server/database/update.js

```diff
@@ -3,7 +3,7 @@ const uuid = require('uuid/v4');
 const hash = require('object-hash');
 const config = require('./config');
 const db = require('./utilities');
-const fetch = require('node-fetch');
+const pcServices = require('./../pcServices');
```

Comment:
what directory does ./../pcServices resolve to?

Reply:
src/server/